src/renderer/config-blocks/SettingsButton.svelte

```diff
@@ -75,28 +94,100 @@
   ];
 
   let suggestionElement = undefined;
+  let minMaxEnabled = false;
+
+  function calculateStepValues(steps, min, max) {
+    const stepValue = Math.floor((max - min) / steps);
+    const res = Array.from(
+      { length: steps },
+      (_, index) => (index + 1) * stepValue
+    );
+    return [0, ...res];
+  }
+
+  let stepValues;
+  $: stepValues = calculateStepValues(
+    Number(bmo),
+    minMaxEnabled ? Number(bmi) : 0,
+    minMaxEnabled ? Number(bma) : 127
+  );
 </script>
 
 <encoder-settings
-  class="{$$props.class} flex flex-col w-full p-2 pointer-events-auto"
+  class="{$$props.class} flex flex-col w-full px-4 py-2 pointer-events-auto"
 >
-  <div class="w-full px-2">
-    <div class="text-gray-500 text-sm pb-1">Button Mode</div>
-    <AtomicInput
-      inputValue={scriptValue}
-      suggestions={suggestions[0]}
-      suggestionTarget={suggestionElement}
-      on:change={(e) => {
-        scriptValue = e.detail;
-      }}
-      validator={(e) => {
-        return new Validator(e).NotEmpty().Result();
-      }}
-      on:validator={(e) => {
-        const data = e.detail;
-        dispatch("validator", data);
-      }}
-    />
+  <div class="text-gray-500 text-sm pb-1">Button Mode</div>
+  <AtomicInput
+    inputValue={bmo}
+    suggestions={suggestions[0]}
+    suggestionTarget={suggestionElement}
+    on:change={(e) => {
+      bmo = e.detail;
+    }}
+    validator={() => {
+      return new Validator().NotEmpty().Result();
+    }}
+    on:validator={(e) => {
+      const data = e.detail;
+      dispatch("validator", data);
+    }}
+  />
+
+  <div class="flex flex-row gap-2">
+    <span class="text-gray-500 text-sm">Step values:</span>
+    <div class="text-white text-sm">
+      {#if Number(bmo) > 1}
+        {#each stepValues as step, i}
+          <span>{step}</span>
+          <span class:hidden={i === stepValues.length - 1} class="mr-2">,</span>
+        {/each}
+      {:else}
+        <span>N/A</span>
+      {/if}
+    </div>
+  </div>
+  <MeltCheckbox bind:target={minMaxEnabled} title={"Enable Min/Max Value"} />
+  <div class="flex flex-row gap-2">
+    <div class="flex flex-col">
+      <span class="text-sm text-gray-500">Max</span>
```

Comment:
That label of the input should be min instead of max!

<span class="text-sm text-gray-500">Min</span>